cutout_cmd/mongo_util.py

```diff
@@ -5,7 +5,7 @@
 from bson.json_util import dumps
 
 
```

Comment:
you mean the one below? connect=False? 
with this arg, Mongo throws errors, from the warnings given, I made this change. With this command, the connection will be initiated at the first operation